packages/tutanota-crypto/test/KyberTest.ts

```diff
@@ -1,12 +1,39 @@
 import o from "@tutao/otest"
-import { loadWasmModuleFromFile } from "./WebAssemblyTestUtils.js"
-import { decapsulate, encapsulate, generateKeyPair } from "../lib/encryption/Liboqs/Kyber.js"
+import { decapsulate, encapsulate, generateKeyPair, LibOQSExports } from "../lib/encryption/Liboqs/Kyber.js"
 import { random } from "../lib/index.js"
-
-const liboqs = await loadWasmModuleFromFile("../lib/encryption/Liboqs/liboqs.wasm")
+import { loadWasmModuleFallback, loadWasmModuleFromFile } from "./WebAssemblyTestUtils.js"
+import { $ } from "zx"
+import fs from "node:fs"
 
 o.spec("Kyber", function () {
+	// We need to generate the webassembly stuff during tests
```

Comment:
Even with comment I don't get it. Don't we also invoke it from the test build script?

Reply:
Tests under packages run before app's test, so no esbuild yet and consequently, no generated wasm or fallback

Reply:
Done